crates/rpc/src/rpc.rs

```diff
@@ -27,6 +27,8 @@ pub trait Rpc {
     fn get_listings(&self) -> Result<Vec<Listing>>;
     #[rpc(name = "getStorefronts")]
     fn get_storefronts(&self) -> Result<Vec<Storefront>>;
+    #[rpc(name = "searchStorefronts")]
+    fn search_storefronts(&self, query: String) -> Result<Vec<Storefront>>;
```

Comment:
Thoughts on moving search to a filter option on get storefronts?

`getStorefronts(search:)`

If search option is set then add the filter doing tsearch.

Reply:
I can look into that, my only concern was whether that would break the API by changing parameter counts.  If it requires a null to be explicitly added to the params list then we can't touch that until we refactor endpoints for v2, but if not I can do it right away.